src/Algebra/Consequences/Propositional.agda

```diff
@@ -83,13 +86,27 @@ module _ {_•_ _◦_ : Op₂ A} (•-comm : Commutative _•_) where
   comm⇒sym[distribˡ] = Base.comm⇒sym[distribˡ] (cong₂ _◦_) •-comm
 
 ------------------------------------------------------------------------
--- Selectivity
+-- Selectivity and Middle-Four Exchange
```

Comment:
I think it would be better to give `Middle-Four Exchange` it's own heading. I'm not sure it should be combined with selectivity?